### ISSUE TITLE
Suggested: Comparison between 2 countries

### ISSUE THREAD

**Issue by yashtatsme92** [ahmadawais/corona-cli#15]
so the basic idea of this implementation would be to have a comparison of statistics between 2 countries.
The command would be `corona -c1 china -c2 italy`.
And the same will result into 2 rows of the said countries and their statistical comparison wherein the higher numbers be indicated by some color.

**Comment by ahmadawais** [comment]
Yes, it's the same thing I suggested you. It's already in motion.

**Comment by dapeng09** [comment]
would be great if there is `corona usa italy` for multiple countries
